addons/mail/static/src/discuss/core/common/discuss_core_common_service.js

```diff
@@ -240,6 +240,13 @@ export class DiscussCoreCommon {
             this.threadService.markAsRead(channel);
         }
         this.env.bus.trigger("discuss.channel/new_message", { channel, message });
+        const authorMember = message.author.channelMembers.find(({ thread }) => thread.eq(channel));
```

Comment:
Nitpicking, but IMO this is more natural:
```js
channel.channelMembers.find(member => member?.persona.eq(message.author));
```
Also if the message has no author, this won't crash. This is not a big deal as all new messages in `discuss.channel` have an author